### PR TITLE
Add rules banner_etc_issue_net and sshd_enable_warning_banner_net

### PR DESCRIPTION
**Pull request by dodys**
#### Description:

- Ubuntu STIG profile has a banner in /etc/issue.net, because of that we created two rules to address this:
banner_etc_issue_net and sshd_enable_warning_banner_net

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/rule.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+documentation_complete: true
+
+prodtype: ubuntu2004
+
+title: 'Modify the System Login Banner'
+
+description: |-
+    To configure the system login banner edit <tt>/etc/issue</tt>. Replace the
```
Comment (jan-cerny) [inline linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/rule.yml:8]:
```suggestion
    To configure the system login banner edit <tt>/etc/issue.net</tt>. Replace the
```

Comment (dodys) [reply linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/rule.yml]: nice catch!
fixed, thanks!

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**